subsys/usb/device/usb_device.c

```diff
@@ -703,9 +784,13 @@ static bool usb_set_interface(struct usb_setup_packet *setup)
 	uint8_t cur_alt_setting = 0xFF;
 	uint8_t cur_iface = 0xFF;
 	bool ret = false;
+	uint32_t ep_bm_active;
+	uint32_t ep_bm;
 
 	LOG_DBG("Set Interface %u alternate %u", setup->wIndex, setup->wValue);
 
+	ep_bm_active = get_ep_bm_for_alt_setting(setup->wIndex, setup->wValue);
```

Comment:
I believe this line refers to `1. First enumerating all the endpoints used across all alternate settings of the chosen interface` in the commit description. However, there is discrepancy between the description and what the code does. I believe the description should be changed to `1. First enumerating all the endpoints used in selected alternate settings of the chosen interface`.

Reply:
Are you perhaps referring to the older function description? The current description for `get_ep_bm_for_alt_setting()` reads as follows:

>Iterate through the USB interface descriptor and accumulate
>a bit-mask representing all the endpoints used by by the specified
>interface and alternate-setting combination. OUT endpoints are
>represented by the lower 16 bit-positions, while the upper 16
>bit-positions indicate IN endpoints

Reply:
No, I am referring to the commit description itself (not to the code): https://github.com/zephyrproject-rtos/zephyr/pull/51824/commits/7f59e76babdcc1a5f7d4d7d12aae9a843d4ebed6
Unfortunately the link will no longer function after you update it due to GitHub review system limitations.

Reply:
Oops - apologies - I changed the commit message and the PR description - please take a look